torch/autograd/grad_mode.py

```diff
@@ -14,7 +14,11 @@ class no_grad(_DecoratorContextManager):
     consumption for computations that would otherwise have `requires_grad=True`.
 
     In this mode, the result of every computation will have
-    `requires_grad=False`, even when the inputs have `requires_grad=True`.
+    `requires_grad=False`, even when the inputs have `requires_grad=True`,
+    UNLESS the result is a tensor created with `requires_grad=True`,
```

Comment:
nit: I think this can be a second sentence. Also mentioning that the exception are 'factory functions' is good: all functions that create a new Tensor and take a requires_grad kwarg.


Also you `requires_grad=True` kwarg is exactly the same as the attribute `requires_grad=True` above. Given that they refer to very different things they shouldn't be the same.